cadc-rest/src/main/java/ca/nrc/cadc/rest/SyncInput.java

```diff
@@ -349,26 +337,21 @@ private void processStream(String name, String contentType, InputStream inputStr
         InlineContentHandler.Content c = inlineContentHandler.accept(name, contentType, new CloseWrapper(inputStream));
         content.put(c.name, c.value);
     }
-    
+
     // this is here to aid in debugging InputStream failures like the one where some library code
     // called close() in a finalise() method, eg during garbage collection
     private class CloseWrapper extends FilterInputStream {
 
-        String tname;
-        public CloseWrapper(InputStream in) {
+        final String tname;
+
+        CloseWrapper(InputStream in) {
             super(in);
             this.tname = "CloseWrapper[" + Thread.currentThread().getName() + "] ";
         }
 
         @Override
         public void close() throws IOException {
```

Comment:
Could you leave these 3 lines in but comment them all out? It took several iterations to figure out what info was needed to debug so I don't want to lose them. thnx

Reply:
Sure.